src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles !== '')
+    .reduce((cssStyles, styles) => {
```

Comment:
The chained methods should have the same indentation to improve readability. Align the `.reduce` method with the other methods above it.